src/Arcus.Security.Core/Storage/Source/SecretStoreBuilderExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
```

Comment:
Let's move this to `Arcus.Security.Core/Extensions`

Reply:
Done